README.md

```diff
@@ -17,11 +19,11 @@ Laptop and Desktop browser for OS X, Windows and Linux.
         # For devs over SSH:
         git clone git@github.com:brave/browser-laptop.git
 
-2. Open the working directory:
+3. Open the working directory:
 
         cd browser-laptop
 
-3. Install the Node (v5+) dependencies:
+4. Install the Node (v5+) dependencies:
 
         npm install
 
```

Comment:
Please have a new section under installation that has the text of your #1 and you can change the Installation section back to normal.